src/components/header-text-with-dropdown/HeaderTextWithDropdown.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'baseline'
+  },
+  input: (disabled: boolean) => ({
+    style: { padding: 0, margin: 0 },
+    disabled: disabled ? true : false
```

Comment:
Ternary is redundant here. You can use just `disabled` instead

Reply:
fixed